temporalio/workflow.py

```diff
@@ -427,6 +427,7 @@ class Info:
     headers: Mapping[str, temporalio.api.common.v1.Payload]
     namespace: str
     parent: Optional[ParentInfo]
+    root: Optional[RootInfo]
```

Comment:
Can we have a test that confirms this is populated (and that it is populated on result of `handle.describe()` from client side)?

Reply:
Added a test in `test_workflow` and added an assertion to a `describe` test in `test_client`